webrender/src/platform/windows/font.rs

```diff
@@ -60,14 +72,56 @@ impl FontContext {
         self.fonts.insert((*font_key).clone(), face);
     }
 
-    fn get_glyph_dimensions_and_maybe_rasterize(&self,
-                                                font_key: FontKey,
-                                                size: Au,
-                                                glyph: u32,
-                                                render_mode: Option<FontRenderMode>)
-                                                -> (Option<GlyphDimensions>, Option<RasterizedGlyph>)
-    {
-        let face = self.fonts.get(&font_key).unwrap();
+    // Assumes RGB format from dwrite, which is 3 bytes per pixel as dwrite
+    // doesn't output an alpha value via GlyphRunAnalysis::CreateAlphaTexture
+    #[allow(dead_code)]
+    fn print_glyph_data(&self, data: &Vec<u8>, width: usize, height: usize) {
+        // Rust doesn't have step_by support on stable :(
+        for i in 0..height {
+            let current_height = i * width * 3;
+
+            for pixel in data[current_height .. current_height + (width * 3)].chunks(3) {
+                let r = pixel[0];
+                let g = pixel[1];
+                let b = pixel[2];
+                print!("({}, {}, {}) ", r, g, b, );
+            }
+            println!("");
+        }
+    }
+
+    fn get_dwrite_texture_type(&self, render_mode: FontRenderMode) ->
```

Comment:
This should be a free function, not part of this struct -- just `fn dwrite_texture_type` in this file.